apps/feedahelp/src/components/ui/features/header/navbar/Navbar.tsx

```diff
@@ -22,9 +18,9 @@ const NewNav = () => {
             avatar={"/assets/avatar.png"}
             className="img-rotate-button"
           >
-            <img className="outer ring" alt="" />
-            <img className="center ring" alt="" />
-            <img className="inner ring" alt="" />
+            <Image src="" className="outer ring" alt="" />
```

Comment:
these will complain for `width` and `height`

Reply:
sir yes I know but I understand how much height and width to give

Reply:
ok we will take care later